crates/kubelet/src/volume/secret.rs

```diff
@@ -0,0 +1,30 @@
+use std::path::PathBuf;
+
+use k8s_openapi::api::core::v1::{KeyToPath, Secret};
+use k8s_openapi::ByteString;
+
+use super::*;
+
+pub(crate) async fn populate(
+    secret: Secret,
+    path: &PathBuf,
+    items: &Option<Vec<KeyToPath>>,
+) -> anyhow::Result<Type> {
+    tokio::fs::create_dir_all(path).await?;
+    let data = secret.data.unwrap_or_default();
+    let data = data.iter().map(|(key, ByteString(data))| async move {
```

Comment:
Nit: I know this is copied code, but if you feel so inclined, could you change this to `into_iter`? We are consuming the data anyway

Reply:
yep!